css/style.css

```diff
@@ -0,0 +1,317 @@
+* {
+	box-sizing:border-box;
+	font-family:arcadeclassic;
+	color:white;
+	cursor:url("../images-logo/cursor-spaceship-medium.png"),auto;
+}
+
+@font-face {
+  font-family: arcadeclassic;
+  src: url("../arcadeclassic/arcadeclassic.ttf");
+}
+
+body {
+	
+	background-image:url("../images-logo/planet2.png");
+	background-size: 2200px 1122px;
+	background-repeat: no-repeat;
+	background-blend-mode: multiply;
+	
+}
+
+/*#index-body-image {
```

Comment:
Why is this section commented?

Reply:
it's the artifact of my first design , I've just delete it and I'm committing now :)